README.md

```diff
@@ -31,42 +31,72 @@
 
 ## Introduction ##
 
-dlisio is an LGPL licensed library for working with well logs in Digital Log
-Interchange Standard (DLIS V1), also known as
-[RP66 V1](http://w3.energistics.org/rp66/v1/Toc/main.html). DLIS V2 is
-out-of-scope for this project, as it is quite different and hardly in use in
-the industry. It is an attempt at a powerful community-driven, portable,
-easy-to-use and flexible library for well logs, that can be used to build a
-wide array of applications.
-
-As of version 0.3.0, dlisio is extended to also read Log Information Standard
-79 ([LIS79](http://w3.energistics.org/LIS/lis-79.pdf)). An extended version of
-the LIS79 standard called LIS84/Enhanced LIS exists, but this version is
-currently not supported by dlisio.
-
-Features are added as they are needed; suggestions, defect reports, and
-contributions of all kinds are very welcome.
+dlisio is an LGPL licensed library for reading well logs in Digital Log
+Interchange Standard (DLIS V1), also known as [RP66
+V1](http://w3.energistics.org/rp66/v1/Toc/main.html) and Log Information
+Standard 79 ([LIS79](http://w3.energistics.org/LIS/lis-79.pdf)).
+
+dlisio is designed as a general purpose library for reading well logs in a
+simple, and easy-to-use manner. Its main focus is making all the data and
+metadata accessible while putting little assumptions on how the data is to be
+used. This makes it suitable as a building block for higher level applications
+as well as being used directly.
+
+dlisio focus above all on correctness, performance and robustness. It's core,
+which does all the heavy lifting, is implemented in C++. Both the C++ core and
+the python wrappers are backed by an extensive test-suite. It strives to be
+robust against files that does not strictly adhere to the specifications, which
+is a widespread issue with both DLIS and LIS files. dlisio tries to account for
+many of the known specification violations out there, but only when it can do
+so without compromising correctness. It will not do any guess work on your
+behalf when such violations pose any ambiguity.
+
+## Installation ##
+
+dlisio supply pre-built python wheels for Windows (32-bit and 64bit), Linux,
+and MacOS for Python 3.6 and above. The wheels are hosted through
+[PyPi](https://pip.pypi.io) and can be installed with:
+
+```bash
+pip install dlisio
+```
+
+See [Build dlisio](#Build-dlisio) for building dlisio from source.
 
 ## Getting started ##
 
-dlisio is in rapid development, and the interfaces are *not* stable. We welcome
-any users and will try our best to accommodate your needs, but we currently make
-no guarantees that code that works today will work tomorrow.
+dlisio's documentation is hosted on
+[readthedocs](https://dlisio.readthedocs.io/en/stable/). Please refer there for
+proper introduction to dlisio, and the file-formats DLIS and LIS. Here is a
```

Comment:
(it's Readme, so I strike for commas, sorry :D )
With comma it sounds like 

> Please refer there for introduction and refer there for the formats

Without comma I would be more inclined to read it as

> introduction to dlisio and introduction to file formats